ServerRecording/README.MD

```diff
@@ -7,82 +7,173 @@ products:
+	    e.added.forEach((addedCall) => {
+	        addedCall.on('stateChanged', (): void => {
+	            if (addedCall.state === 'Connected') {
+	                addedCall.info.getServerCallId().then(result => {
+	                    dispatch(setServerCallId(result));
+	                }).catch(err => {
+	                    console.log(err);
+	                });
+	            }
+	        });
+	    });
+	});
+	```
+
+ For complete details on how to build a client application using ACS calling SDK, refer to https://docs.microsoft.com/azure/communication-services/samples/calling-hero-sample?pivots=platform-web
 
 ### Troubleshooting
 
-1. Solution doesn\'t build, it throws errors during NPM installation/build
+1. Solution doesn\'t build / It throws errors during MVN installation/build
 
-	- Check if all the config keys are present
+	-  Check if all the config keys are present, and rebuild with `mvn package`, then `mvn clean install`
 
-	- Run mvn package, then mvn clean install
+	- After installing the JDK and building, if you see "invalid target release: 11", verify that your JAVA_HOME variable does in fact point to your Java 11 installation (as opposed to a previous installation).
 
 ## Publish to Azure
 
-1. Build java code using : mvn clean install  
+1. Build java code :
+
+	```bash
+	mvn clean install  
+	```
+
+1. Run app locally using :
+
+	```bash
+	mvn spring-boot:run
+ 	```
+
+1. Configure Maven to build image to your Azure Container Registry.
+
+ 	- Navigate to the completed project directory for your Spring Boot application and open the pom.xml file with a text editor.
+ 	- Update the <properties> collection in the pom.xml file with the latest version of [jib-maven-plugin](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin), login server value, and access settings for your Azure Container Registry created using [Create container registry](https://docs.microsoft.com/azure/developer/java/spring-framework/deploy-spring-boot-java-app-on-linux#create-an-azure-container-registry-to-use-as-a-private-docker-registry). For example:
+
+		```xml
+		<properties>
+			<jib-maven-plugin.version>2.5.2</jib-maven-plugin.version>
+			<docker.image.prefix>{docker.image.prefix}</docker.image.prefix>
+			<java.version>1.8</java.version>
+		</properties>
+		```
+  	- Add [jib-maven-plugin](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin) to the <plugins> collection in the pom.xml file. This example uses version 2.5.2.
+
+		Specify the base image in ```<image>``` tag under ```<from>```, here ```mcr.microsoft.com/java/jre:11-zulu-alpine```. Specify the name of the final image to be built from the base in ```<image>``` under ```<to>```.
+
+		Authentication {docker.image.prefix} is the Login server of Azure Container Registry. The {project.artifactId} is the name and version number of the JAR file from the first Maven build of the project.
```

Comment:
The first word in this line "Authentication", is that a typo?

Reply:
Terminology used from Microsoft docs, Probably a typo. Corrected.